boards/arm/stm32f3_disco/stm32f3_disco_E.yaml

```diff
@@ -0,0 +1,22 @@
+identifier: stm32f3_disco@E
```

Comment:
stm32f3_disco_E

Reply:
Alright, will do, did not know you can use underscore also there!